src/legacy/ui/field_formats/mixin/field_formats_service.test.ts

```diff
@@ -39,7 +36,7 @@ describe('FieldFormatsService', function() {
   let fieldFormatsService: FieldFormatsService;
 
   beforeEach(function() {
-    const fieldFormatClasses = [createNumberFormat(FieldFormat)];
+    const fieldFormatClasses = [createNumberFormat()] as any;
```

Comment:
Can we avoid of using 'any' here?  I think createNumberFormat method should return instance of FieldFormat

Reply:
please no `any`s

Reply:
Done